packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -161,7 +162,7 @@ function MapsIndoorsMap({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId
         }
     }, [locationId]);
 
-    /*
+    /**
```

Comment:
I realize now that this is just a function call, and not a function definition. Thus is should not be a JSDoc block but just a regular comment. I believe we've used JSDoc blocks a lot of other places too where it wasn't relevant.

```suggestion
    /*
```